README.md

```diff
@@ -178,9 +178,12 @@ Options are specified with the -D flag.  The allowed options include:
     BUILD_CLI_EXECUTABLES=(ON/OFF): whether or not to build command-line programs
     BUILD_PYTHON_BINDINGS=(ON/OFF): whether or not to build Python bindings
     PYTHON_EXECUTABLE=(/path/to/python_version): Path to specific Python executable
+    BUILD_JULIA_BINDINGS=(ON/OFF): whether or not to build Julia bindings
+    JULIA_EXECUTABLE=(/path/to/julia): Path to specific Julia executable
     BUILD_TESTS=(ON/OFF): whether or not to build tests
     BUILD_SHARED_LIBS=(ON/OFF): compile shared libraries as opposed to
        static libraries
+    DISABLE_DOWNLOADS=(ON/OFF): whether to disable all downloads during build
     DOWNLOAD_ENSMALLEN=(ON/OFF): If ensmallen is not found, download it
     DOWNLOAD_STB_IMAGE=(ON/OFF): If STB is not found, download it
     ENSMALLEN_INCLUDE_DIR=(/path/to/ensmallen/include): path to include directory
```

Comment:
Okay, just one more doubt.
I think we should mention about `STB_IMAGE_INCLUDE_DIR` like we have mentioned about `ENSMALLEN_INCLUDE_DIR` over here to change the `image search` location. Let me know if I am going wrong. There is a high probability that I can go wrong.

Reply:
But I think this should be done.

Reply:
Ah, agreed, this is a good point.  I've added documentation wherever `ENSMALLEN_INCLUDE_DIR` was mentioned in d0a2b8fb. :+1: